mtgsqlive/json2sql.py

```diff
@@ -46,9 +45,14 @@ def validate_io_streams(input_file: pathlib.Path, output_file: pathlib.Path) ->
     :param output_file: Output file (SQLite)
     :return: Good to continue status
     """
-    if not input_file.is_file():
-        LOGGER.fatal("Input file {} does not exist.".format(input_file))
-        return False
+    if input_file.is_file():
+        LOGGER.info("Building using AllSets.json master file.")
+    else:
```

Comment:
if/elif/else instead of nested if/else

Reply:
ha yeah, I used elif later in the parse_and_import_cards function so I definitely know to use it, but my brain doesn't always do what I want. Thanks for catching it.